extensions/wikia/ExactTargetUpdates/client/builders/CreateRequestBuilder.class.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Wikia\ExactTarget\Builders;
+
+class CreateRequestBuilder extends BaseRequestBuilder {
+	private static $supportedTypes = [ self::GROUP_TYPE, self::SUBSCRIBER_TYPE ];
+
+	public function build() {
+		$type = self::DATA_EXTENSION_OBJECT_TYPE;
+
+		if ( $this->type === self::GROUP_TYPE ) {
```

Comment:
maybe switch?
http://www.phpbench.com/ it's almost same in concern of benchmarks and it will increase readability.